userdocs/creating-a-codelist-from-scratch.md

```diff
@@ -90,3 +90,8 @@ information on procedures for building codelists to use in OpenSAFELY research,
 Once the codelist is reviewed, it can be published, using the __Publish version__ link from the codelist's homepage.  Publishing a codelist version will make that version permanent, and will delete any other draft or in-review versions.
 
         publish.png
+        
+### Selecting appropriate codes for your codelist
+
+Choosing which codes to include in your codelist can be challenging without understanding their usage in clinical practice. Some clinical activities are represented by a single code, while others may require a comprehensive list of codes to accurately capture the intended clinical activity. Helpfully, [NHS Digital provides a dataset on SNOMED CT code usage in primary care](https://digital.nhs.uk/data-and-information/publications/statistical/mi-snomed-code-usage-in-primary-care), available since 2011. This dataset includes annual counts of how often each SNOMED CT code is recorded in GP patient records across England. When building a SNOMED CT codelist, this may help you decide whether to include or exclude a code. You can explore recorded usage of individual codes or entire codelists, including those on OpenCodelists, using [this prototype SNOMED CT code usage explorer](https://snomed-code-usage.streamlit.app/).
```

Comment:
maybe overly pedantic, but I don't think the file has been available since 2011!

Reply:
Yeh you're right, I'll fix it